package.json

```diff
@@ -8,7 +8,6 @@
   "typings": "dist/main/index.d.ts",
   "repository": "https://github.com/Sphereon-Opensource/pex.git",
   "license": "Apache-2.0",
-  "packageManager": "pnpm@8.15.9+sha256.daa27a0b541bc635323ff96c2ded995467ff9fe6d69ff67021558aa9ad9dcc36",
```

Comment:
Can we keep this? It makes it easier to switch between different package managers for different projects :) 

`pnpm/action-setup` will also read from this value. 

You can do `corepack use pnpm@9.4.0` to update the value.

See https://nodejs.org/api/corepack.html